sphinx_immaterial/apidoc/python/attribute_style.py

```diff
@@ -17,7 +18,12 @@ def handle_signature(
         typ = self.options.get("type")
         if typ:
             signode += sphinx.addnodes.desc_sig_punctuation("", " : ")
-            signode += sphinx.domains.python._parse_annotation(typ, self.env)
+            if sphinx.version_info >= (7, 3):
+                signode += sphinx.domains.python._annotations._parse_annotation(  # type: ignore[attr-defined]
```

Comment:
For these it would probably be cleaner to just handle the change by importing parse_annotations directly and doing the version check there.

Reply:
done